### PR TITLE
[Enhancement] Method invocation 'getBytes' may produce 'NullPointerException' [HttpCommand]

### PR DESCRIPTION
**Pull request by weihubeats**
Fixes https://github.com/apache/incubator-eventmesh/issues/2539

### Motivation

*Explain the content here.*
*Explain why you want to make the changes and what problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/http/HttpCommand.java [hunk 1]
```diff
@@ -110,132 +114,62 @@ public HttpCommand createHttpCommandResponse(EventMeshRetCode eventMeshRetCode)
...
-        return sb.toString();
+        return "httpCommand={"
+            + cmdType
+            + ","
+            + httpMethod
+            + "/" + httpVersion
+            + ","
+            + "requestCode="
+            + requestCode
+            + "}";
```
Comment (walterlife) [inline eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/http/HttpCommand.java:164]: It is recommended to use StringBuilder uniformly here to ensure the consistency of the internal code of the class as much as possible.

Comment (weihubeats) [reply eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/http/HttpCommand.java]: Can also, with + just more brief, there is no performance gap between the two

Comment (walterlife) [reply eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/http/HttpCommand.java]: With no performance or code style concerns, the author should maintain consistency with the existing code.
For more details, please read the google code review guide.
https://google.github.io/eng-practices/review/reviewer/looking-for.html#consistency

**Review by walterlife — APPROVED** [review]
LGTM

**Pull request merged**